dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/ActivityName.ts

```diff
@@ -0,0 +1,122 @@
+  ElectricityGenerationFromWindPower = "Electricity generation from wind power",
+  ElectricityGenerationFromOceanEnergyTechnologies = "Electricity generation from ocean energy technologies",
+  ElectricityGenerationFromHydropower = "Electricity generation from hydropower",
+  ElectricityGenerationFromGeothermalEnergy = "Electricity generation from geothermal energy",
+  ElectricityGenerationFromRenewableNonFossilGaseousAndLiquidFuels = "Electricity generation from renewable non-fossil gaseous and liquid fuels",
+  ElectricityGenerationFromBioenergy = "Electricity generation from bioenergy",
+  TransmissionAndDistributionOfElectricity = "Transmission and distribution of electricity",
+  StorageOfElectricity = "Storage of electricity",
+  StorageOfThermalEnergy = "Storage of thermal energy",
+  StorageOfHydrogen = "Storage of hydrogen",
+  ManufactureOfBiogasAndBiofuelsForUseInTransportAndOfBioliquids = "Manufacture of biogas and biofuels for use in transport and of bioliquids",
+  TransmissionAndDistributionNetworksForRenewableAndLowCarbonGases = "Transmission and distribution networks for renewable and low-carbon gases",
+  DistrictHeatingCoolingDistribution = "District heating/cooling distribution",
+  InstallationAndOperationOfElectricHeatPumps = "Installation and operation of electric heat pumps",
+  CogenerationOfHeatCoolAndPowerFromSolarEnergy = "Cogeneration of heat/cool and power from solar energy",
+  CogenerationOfHeatCoolAndPowerFromGeothermalEnergy = "Cogeneration of heat/cool and power from geothermal energy",
+  CogenerationOfHeatCoolAndPowerFromRenewableNonFossilGaseousAndLiquidFuels = "Cogeneration of heat/cool and power from renewable non-fossil gaseous and liquid fuels",
+  CogenerationOfHeatCoolAndPowerFromBioenergy = "Cogeneration of heat/cool and power from bioenergy",
+  ProductionOfHeatCoolFromSolarThermalHeating = "Production of heat/cool from solar thermal heating",
+  ProductionOfHeatCoolFromGeothermalEnergy = "Production of heat/cool from geothermal energy",
+  ProductionOfHeatCoolFromRenewableNonFossilGaseousAndLiquidFuels = "Production of heat/cool from renewable non-fossil gaseous and liquid fuels",
+  ProductionOfHeatCoolFromBioenergy = "Production of heat/cool from bioenergy",
+  ProductionOfHeatCoolUsingWasteHeat = "Production of heat/cool using waste heat",
+  PreCommercialStagesOfAdvancedTechnologiesToProduceEnergyFromNuclearProcessesWithMinimalWasteFromTheFuelCycle = "Pre-commercial stages of advanced technologies to produce energy from nuclear processes with minimal waste from the fuel cycle",
+  ConstructionAndSafeOperationOfNewNuclearPowerPlantsForTheGenerationOfElectricityAndOrHeatIncludingForHydrogenProductionUsingBestAvailableTechnologies = "Construction and safe operation of new nuclear power plants, for the generation of electricity and/or heat, including for hydrogen production, using best-available technologies",
```

Comment:
Note for myself.
Check if this overlaps with ActivityTree in the upload page branch

Reply:
This will be obsolete once merged into the feature branch with the upload page